src/anniversary/cri/libadxe/adx_fcch.c

```diff
@@ -0,0 +1,7 @@
+#include "cri/private/libadxe/adx_fcch.h"
+#include "common.h"
+#include "sdk/eekernel.h"
```

Comment:
SDK and CRI headers should be included with angle brackets I think
```suggestion
#include <eekernel.h>
```